softlayer/softlayer_product_package_service.go

```diff
@@ -9,6 +9,8 @@ type SoftLayer_Product_Package_Service interface {
 
 	GetItemPrices(packageId int) ([]datatypes.SoftLayer_Product_Item_Price, error)
 	GetItemPricesBySize(packageId int, size int) ([]datatypes.SoftLayer_Product_Item_Price, error)
+	GetItemPricesBySizeAndIops(packageId int, size int, capacity int) ([]datatypes.SoftLayer_Product_Item_Price, error)
```

Comment:
@maximilien ,
I think it should be a complex service method in the `SoftLayer_Product_Package`. The based method is [getItemPrices](http://sldn.softlayer.com/reference/services/SoftLayer_Product_Package/getItemPrices), what we do is to customize the method with result filter and mask. The following SL blog is the example we followed.
https://softlayer.github.io/python/place_order_block_storage_performance_iscsi/